### PR TITLE
Update Cloud SQL Postgres sample to create table on app start

### LINKED ISSUE #2306: Cloud SQL Postgres sample requires user run a createTable.js script
Issue: The Cloud SQL Postgres sample requires the user to first run a createTable.js script (in order to create the sample votes table) before running the sample itself.

Expected: Running the sample should create the sample table required to run the sample similarly to how the MySQL sample does it.

Sample location:
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/master/cloud-sql/postgres/knex

### PR DESCRIPTION
**Pull request by jsimonweb**
Fixes #2306

### CODE REVIEW

File: cloud-sql/postgres/knex/server.js [hunk 1]
```diff
@@ -133,6 +146,29 @@ const createPool = () => {
...
+const ensureSchema = async pool => {
+  const hasTable = await pool.schema.hasTable('votes');
+  if (!hasTable) {
+    return pool.schema.createTable('votes', table => {
+      table.increments('vote_id').primary();
+      table.timestamp('time_cast', 30).notNullable();
+      table.specificType('candidate', 'CHAR(6)').notNullable();
+    });
+  }
+  console.log("Ensured that table 'votes' exists");
```
Comment (enocom) [inline cloud-sql/postgres/knex/server.js:158]: This should probably be `logger.info` or whatever is appropriate for winston.

**Comment by bcoe** [comment]
@jsimonweb looks good to me, but there do appear to be two failing links when tests run:

Error: [404] https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/update-cloud-sql-postgres-sample/appengine
Error: [404] https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/update-cloud-sql-postgres-sample/CONTRIBUTING.md

Any idea what's referencing these, such that we might fix the links?

**Comment by jsimonweb** [comment]
@bcoe 
>>Any idea what's referencing these, such that we might fix the links?

Looks like these links might have been caused by the recent change to rename the repo's main branch and not a result of this PR.

Merged branch 'main' into update-cloud-sql-postgres-sample branch to sync PR with 'main'.

**Comment by sofisl** [comment]
Given that we've merged this: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/2314#pullrequestreview-732216328

I think we need to make the Kokoro config changes first so that tests can run on this PR. I'll make sure to make any necessary to changes to this PR for it to pass.

**Review by JustinBeckwith — APPROVED** [review]

**Pull request merged**